cmake/vcpkghelpers.cmake

```diff
@@ -87,70 +92,70 @@ function(ivw_vcpkg_install name)
 	endif()
 
     set(options EXT)
-    set(oneValueArgs OUT_COPYRIGHT OUT_VERSION COPYRIGHT MODULE)
+    set(oneValueArgs OUT_COPYRIGHT OUT_VERSION MODULE)
     set(multiValueArgs "")
     cmake_parse_arguments(ARG "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
 	string(TOLOWER "${name}" lowercase_name)
-	set(pkgdir "${_VCPKG_ROOT_DIR}/packages/${lowercase_name}_${VCPKG_TARGET_TRIPLET}")
-    set(portdir "${_VCPKG_ROOT_DIR}/ports/${lowercase_name}")
-
-    if(EXISTS "${pkgdir}/bin")
-        install(
-    	   DIRECTORY "${pkgdir}/bin/" 
-    	   DESTINATION ${IVW_RUNTIME_INSTALL_DIR}
-           COMPONENT Application
-    	   FILES_MATCHING PATTERN "*${CMAKE_SHARED_LIBRARY_SUFFIX}"
-        )
-        install(
-           DIRECTORY "${pkgdir}/bin/" 
-           DESTINATION ${IVW_RUNTIME_INSTALL_DIR}
-           COMPONENT Development
-           FILES_MATCHING PATTERN "*.pdb"
-        )
-        install(
-           DIRECTORY "${pkgdir}/lib/" 
-           DESTINATION ${IVW_LIBRARY_INSTALL_DIR}
-           COMPONENT Development
-           FILES_MATCHING PATTERN "*${CMAKE_LINK_LIBRARY_SUFFIX}"
-        )
-    endif()
 
-    set(version "?.?.?")
-    set(depends "")
-    if(EXISTS ${pkgdir}/CONTROL)
-        file(STRINGS ${pkgdir}/CONTROL lines)
-        foreach(line IN LISTS lines)
-            if(line MATCHES "Version: (.*)")
-                set(version ${CMAKE_MATCH_1})
-            endif()
-            if(line MATCHES "Depends: (.*)")
-                set(depDepends ${CMAKE_MATCH_1})
-                string(REPLACE "," ";" depDepends ${depDepends})
-                list(TRANSFORM depDepends STRIP)
-                list(APPEND depends ${depDepends})
-            endif()
-        endforeach()
+    if(NOT DEFINED ivw_vcpkg_info_${lowercase_name})
+        message(STATUS "Vcpkg fetching metadata for: ${name}")
+        execute_process(
+            COMMAND "${PYTHON_EXECUTABLE}" "${IVW_TOOLS_DIR}/vcpkginfo.py"
+                --vcpkg "${_VCPKG_EXECUTABLE}" 
+                --pkg ${lowercase_name}
+                --triplet ${VCPKG_TARGET_TRIPLET}
+            OUTPUT_VARIABLE pkgInfo
+            OUTPUT_STRIP_TRAILING_WHITESPACE
+        )
```

Comment:
Should verify that pkgInfo contains expected output? Not empty for example

Reply:
will update this when my changes to CMakes lands in 3.19